src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.jsx

```diff
@@ -337,5 +352,65 @@ describe('VAOS <RequestedAppointmentDetailsPage>', () => {
         name: 'We’re sorry. We’ve run into a problem',
       }),
     ).to.be.ok;
+
+    expect(document.activeElement).to.have.tagName('h1');
+  });
+
+  it('should display pending document title', async () => {
+    // Verify VA pending
+    mockSingleRequestFetch({
+      request: appointment,
+      type: 'va',
+    });
+
+    renderWithStoreAndRouter(<AppointmentList />, {
+      initialState,
+      path: `/requests/${appointment.id}`,
+    });
+
+    await waitFor(() => {
+      expect(global.document.title).to.equal(
+        `Pending VA primary care appointment`,
+      );
+    });
+
+    // Verify CC pending appointment
+    mockSingleRequestFetch({
+      request: ccAppointmentRequest,
+      type: 'cc',
+    });
+
+    renderWithStoreAndRouter(<AppointmentList />, {
+      initialState,
+      path: `/requests/${appointment.id}`,
+    });
+
+    await waitFor(() => {
+      expect(global.document.title).to.equal(
+        `Pending Community care audiology (hearing aid support) appointment`,
+      );
+    });
+  });
+
+  it('should display cancel document title', async () => {
+    mockSingleRequestFetch({
+      request: appointment,
+      type: 'va',
+    });
+    // Verify cancel VA appt
+    appointment.attributes = {
+      ...appointment.attributes,
+      status: 'Cancelled',
```

Comment:
@jbalboni, As soon as I introduce the `Canceled` status into the appointment attribute in this last test, it is over riding the previous `Submitted` status and causing the unit test to fail. I thought having the below code at the start of the `describe` resets the data for every `it` description test?

```
beforeEach(() => {
    mockFetch();
    MockDate.set(testDate);
  });

  afterEach(() => {
    resetFetch();
    MockDate.reset();
```

Reply:
No, it just resets the fetch mock itself, meaning it removes all of the urls we set up to return mock data from. This test modifies data that's shared between all the tests. If that data was created inside a `beforeEach` your test would work.

The easiest thing is probably to create your own copy in this test like:

```
const canceledAppointment = { ...appointment };
canceledAppointment.attributes = {
   ...canceledAppointment.attributes,
   status: 'Cancelled',
};
mockSingleRequestFetch({
   request: canceledAppointment,
   type: 'va',
});
```

Reply:
Took the easy route and created a copy of data then modify the `status` attribute just for the cancel test.